common/management/commands/celery.py

```diff
@@ -0,0 +1,22 @@
+# pylint: skip-file
+import shlex
+from subprocess import PIPE  # nosec
+
+from django.core.management.base import BaseCommand
+from django.utils import autoreload
+
+import psutil
+
+
+def restart_celery():
+    for proc in psutil.process_iter():
+        if proc.name() == 'celery':
+            proc.kill()
```

Comment:
Does this cause a warm shutdown?

Reply:
It does sends `SIGKILL` instead, I'll change that to send a `SIGTERM` which will cause a warm shutdown

Reply:
Make sure this will wait for Celery to dies. If not, you may need to watch for the process to end.

Reply:
I resolved this by send a `SIGTERM` only to the parent process.